src/js/user-profile/components/AuthApplicationSection.jsx

```diff
@@ -1,60 +1,111 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+const contentConditions = [
+  [
+    <p key="hca"><a href="/health-care/apply/">Apply for health care</a></p>,
+    ['hca']
+  ],
+  [
+    (<p key="edu-benefits"><a href="/education/apply-for-education-benefits">
+      Apply for Education Benefits
+    </a></p>),
+    ['edu-benefits']
+  ],
+  [
+    <p key="discharge-upgrade"><a href="/discharge-upgrade-instructions/">Apply for a discharge upgrade if needed</a></p>,
+    null
+  ],
+  [
+    <p key="rx"><a href="/health-care/prescriptions">Refill your prescription</a></p>,
+    ['rx']
+  ],
+  [
+    <p key="messaging"><a href="/health-care/messaging">Message your health care team</a></p>,
+    ['messaging']
+  ],
+  [
+    <p key="health-records"><a href="/health-care/health-records">Get your VA health records</a></p>,
+    ['health-records']
+  ],
+  [
+    <p key="claims"><a href="/track-claims">Check your claim and appeal status</a></p>,
+    ['evss-claims', 'appeals-status']
+  ],
+  [
+    (<p key="post-911"><a href="/education/gi-bill/post-9-11/ch-33-benefit">
+      Get your Post-9/11 GI Bill statement of benefits
+    </a></p>),
+    ['evss-claims']
+  ],
+  [
+    <p key="vic"><a href="/veteran-id-card">Apply for a Veteran ID Card</a></p>,
+    ['id-card']
+  ]
+];
+
 class AuthApplicationSection extends React.Component {
   render() {
-    let content;
-    const verifiedAccountType = 3;// verified ID.me accounts are type 3
-    const isVerifiedUser = this.props.userProfile.accountType === verifiedAccountType;
-    const idCardAvailable = this.props.userProfile.services.indexOf('id-card') !== -1;
-    if (isVerifiedUser) {
-      content = (
+    const { services } = this.props.userProfile;
+    const availableServices = [];
+    const unavailableServices = [];
+    const isAvailable = services.reduce((acc, service) => {
```

Comment:
This might be clearer if you convert it into a Set instead of an object:

`const isAvailable = new Set(services);`

Then below you can do `isAvailable.has(service)`. And like Ben says above, coming up with a different name than `isAvailable` would help, too.